sytest/pipeline.yml

```diff
@@ -185,7 +185,7 @@ steps:
         - exit_status: 2
           limit: 2
 
-  - label: "Dendrite - :go: 1.11 / :postgres: 9.6"
+  - label: "Dendrite - :go: 1.13 / :postgres: 9.6"
```

Comment:
I'm guessing the `matrixdotorg/sytest-dendrite` image was updated a while ago and this is just updating the build name to reflect reality?

Reply:
Yep indeed, we've been requiring Go 1.13 for some time now.